requirements-test.txt

```diff
@@ -1,5 +1,6 @@
-nose==1.3.4
+docker-py==1.8.1
```

Comment:
I also do not think I am using the docker library here, so we can just take it out